### PR TITLE
feat(consumer): add a new option

### PR DESCRIPTION
**Pull request by alecgerona**
config can be set to conditionally ack the received messages

### CODE REVIEW

File: pyrmq/consumer.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def __init__(
         :keyword retry_queue_suffix: The suffix that will be appended to the ``queue_name`` to act as the name of the retry_queue. Default: ``retry``
         :keyword max_retries: Number of maximum retries for DLK retry logic. Default: ``20``
         :keyword queue_args: Your queue arguments. Default: ``None``
+        :keyword auto_ack: Flag whether to ack or nack the consumed message regardless of its outcome. Default: ``True``
```
Comment (sibayanjasper) [inline pyrmq/consumer.py:58]: `Flag whether to ack or nack` but we never nack.

**Review by sibayanjasper — CHANGES_REQUESTED** [review]

**Comment by sibayanjasper** [comment]
@alecgerona why is the PR description like that?

**Comment by sibayanjasper** [comment]
> @alecgerona why is the PR description like that?

oh it's from the commit message
maybe avoid using long commit message on the first line
use something like a short commit title on the first line and then short/long explanation/description from the 3rd line onwards

**Comment by sibayanjasper** [comment]
> > @alecgerona why is the PR description like that?
> 
> oh it's from the commit message
> maybe avoid using long commit message on the first line
> use something like a short commit title on the first line and then short/long explanation/description from the 3rd line onwards

if you're using git commit in terminal and use vim (or any editor) for the message, it will tell you that the first line is too long

**Comment by alecgerona** [comment]
> > @alecgerona why is the PR description like that?
> 
> oh it's from the commit message
> maybe avoid using long commit message on the first line
> use something like a short commit title on the first line and then short/long explanation/description from the 3rd line onwards
> 
> if you're using git commit in terminal and use vim (or any editor) for the message, it will tell you that the first line is too long

Yeah I'm using commitizen. I'll fix this commit. There's an option for an extended commit description there. Just wasn't sure that it would display it like this tho.

**Review by sibayanjasper — APPROVED** [review]

**Pull request merged**